PSScriptAnalyzerRules.psd1

```diff
@@ -0,0 +1,84 @@
+@{
+    Severity     = @('Error')
+    IncludeRules = @(
+        'PSUseCompatibleSyntax',
+        'PSAvoidUsingCmdletAliases',
+        'PSAvoidUsingWriteHost',
+        'PSAvoidDefaultValueSwitchParameter',
+        'PSReservedCmdletChar',
+        'PSReservedParams',
+        'PSAvoidUsingUserNameAndPassWordParams',
+        'PSAvoidUsingPlaintTextForPassword',
+        'PSAvoidUsingWMICmdlet',
+        'PSAvoidUsingWriteHost',
+        'PSMisleadingBacktick',
+        'PSMissingModuleManifestField',
+        'PSPossibleIncorrectComparisonWithNull',
+        'PSUseApprovedVerbs',
+        'PSUseOutputTypeCorrectly',
+        'PSShouldProcess',
+        'PSUserToExportFieldsInManifest',
+        'PSUseSingularNouns',
+        'PSAvoidUsingInvokeExpression',
+        'PSUseShouldProcessForStateChangingFunctions',
+        'PSUseDeclaredVarsMoreThanAssignments',
+        'PSUseCore',
+        'PSPlaceOpenBrace',
+        'PSPlaceCloseBrace',
+        'PSUseConsistentWhitespace',
+        'PSUseConsistentIndentation',
+        'PSAlignAssignmentStatement'
+        # 'PSUseCorrectCasing' ## We do not like the use of -EQ, prefer -eq
+    )
+    Rules        = @{
+        PSUseCompatibleSyntax      = @{
+            Enable        = $true
+            TargetVersion = @(
+                '3.0',
+                '4.0',
+                '5.1',
+                '6.2'
+            )
```

Comment:
PSOpenAI only supports Windows PowerShell 5.1 and PowerShell 7.x
Compatibility with pre-4.0 and 6.x is not a consideration.
I don't know how to set this parameter correctly in PSScriptAnalyzer, is this ok?

```suggestion
            TargetVersion = @(
                '5.1',
                '7.2'
            )
```

Reply:
Seems reasonable! I just copy/pasted from my other project and didn't even realize what this was for 😅  I dig it.